Code/Components/Player/Player.cs

```diff
@@ -59,13 +61,32 @@ protected override void OnUpdate()
 		}
 
 		// Crosshair
+		DrawCrosshair();
+	}
+
+	/// <summary>
+	/// Draws the player's crosshair based on what they've defined
+	/// </summary>
+	private void DrawCrosshair()
+	{
```

Comment:
Just noticed this from MP testing:
```suggestion
	{
		if ( IsProxy ) return;
```